src/test/java/catalinazoldi/tests/Tema9/Tema7/CURAHealthcare.java

```diff
@@ -0,0 +1,115 @@
+package catalinazoldi.tests.Tema9.Tema7;
+
+import catalinazoldi.tests.Tema7.BaseTest;
```

Comment:
Replace this import with the one from Tema9 package -> `import catalinazoldi.tests.Tema9.Tema7.BaseTest;`